proto/qdrant/cloud/account/v1/account.proto

```diff
@@ -435,8 +435,13 @@ message Account {
 // An AccountInvite represents an invitation for a user to join an account.
 message AccountInvite {
   // Unique identifier for the invite (in GUID format).
-  // This is a read-only field, generated by the server.
-  string id = 1 [(buf.validate.field).string = {uuid: true}];
+  // This is a read-only field and will be available after an invite is created.
```

Comment:
I also updated the comment to make it more consistent with the other messages in our API.